src/com/mani/lang/core/Parser.java

```diff
@@ -270,13 +272,22 @@ private Stmt switchStatement() {
         consume(TokenType.LEFT_BRACE, "Expect '{' after switch condition");
         while (! match(TokenType.RIGHT_BRACE)) {
             if (match(TokenType.CASE)) {
+                if (seenDefault) {
+                    throw new Return("Default is already declared before.");
```

Comment:
Please change this to `Default has already been declared.`

Reply:
@crazywolf132 Addressed in c098248